pkg/cmd/diagnose.go

```diff
@@ -44,60 +87,135 @@ func (o *DiagnoseOptions) Run() error {
 	}
 
 	log.Logger().Infof("Running in namespace: %s", util.ColorInfo(ns))
-	if o.showOption("version") {
-		err := printStatus(o, "Jenkins X Version", "jx", "version", "--no-version-check")
-		if err != nil {
-			return err
-		}
+	_, table := o.GetPackageVersions(ns, o.HelmTLS)
+
+	// os version
+	osVersion, err := o.GetOsVersion()
+	if err != nil {
+		log.Logger().Warnf("Failed to get OS version: %s", err)
+	} else {
+		table.AddRow("Operating System", util.ColorInfo(osVersion))
 	}
 
-	if o.showOption("status") {
-		err := printStatus(o, "Jenkins X Status", "jx", "status")
-		if err != nil {
-			return err
+	table.Render()
+
+	switch len(o.Args) {
+	//ToDo: This wont be required after the deprecated --show flag is removed
+	case 0:
+		if o.showOption("version") {
+			err := version.NewCmdVersion(o.CommonOptions).Execute()
+			if err != nil {
+				return err
+			}
 		}
-	}
 
-	if o.showOption("pvc") {
-		err := printStatus(o, "Kubernetes PVCs", "kubectl", "get", "pvc", "--namespace", ns)
-		if err != nil {
-			return err
+		if o.showOption("status") {
```

Comment:
```jx diagnose --show=status
Flag --show has been deprecated, use jx diagnose <object> instead. This will be removed on July 1, 2020
Running in namespace: jx
NAME                          VERSION
jx                            2.1.60-dev+83850f2b5
Kubernetes cluster            v1.14.10-gke.36
kubectl (installed in JX_BIN) v1.13.2
helm client                   2.12.2
git                           2.20.1
Operating System              Mac OS X 10.14.6 build 18G4032
Error: unknown flag: --show
Usage:
  status [node] [flags]

Aliases:
  status, status

Examples:
  # displays the current status of the Kubernetes cluster
  jx status

Flags:
  -h, --help          help for status
  -n, --node string   the named node to get

error: unknown flag: --show
```

Reply:
For pods and ingresses it works!?